subsys/nrf_security/Kconfig.tls

```diff
@@ -325,6 +345,29 @@ config MBEDTLS_SSL_CIPHERSUITES
 	  Warning: This field has offers no validation checks.
 	  MBEDTLS_SSL_CIPHERSUITES setting in mbed TLS config file.
 
+if MBEDTLS_TLS_VERSION_1_3
+
+config MBEDTLS_SSL_PROTO_TLS1_3
+	bool "Enable TLS version 1.3 protocol"
```

Comment:
Is there any reason to turn `MBEDTLS_SSL_PROTO_TLS1_3` off if `MBEDTLS_TLS_VERSION_1_3` is enabled?

Reply:
No it should be by default enabled

Reply:
If no, then why not make this symbol [promptless](https://docs.zephyrproject.org/4.0.0/build/kconfig/tips.html#:~:text=Only%20put%20a%20prompt)?